types/scrambo/scrambo-tests.ts

```diff
@@ -0,0 +1,11 @@
+import Scrambo from 'scrambo';
```

Comment:
Use the commonjs syntax `import Scrambo = require('scrambo')`.

Consumers of the package who want to use `import Scrambo from 'scrambo'` can add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases, so must have `"esModuleInterop": false`.